src/os/ui/filter/op/istrueop.js

```diff
@@ -0,0 +1,70 @@
+goog.provide('os.ui.filter.op.IsTrue');
+
+goog.require('os.ui.filter.op.Op');
+goog.require('os.xsd.DataType');
+
+
+/**
+ * A 'PropertyIsTrue' operation class.
+ * Based on the OGC Filter Spec
+ *
+ * @extends {os.ui.filter.op.Op}
+ * @constructor
+ */
+os.ui.filter.op.IsTrue = function() {
+  os.ui.filter.op.IsTrue.base(this, 'constructor',
+      'And', 'is true', 'true',
+      [os.xsd.DataType.BOOLEAN, os.xsd.DataType.INTEGER, os.xsd.DataType.STRING],
+      'hint="os.ui.filter.op.IsTrue"', 'Supports true, 1, and "true" (case insensitive)', 'span', true);
+  this.matchHint = 'os.ui.filter.op.IsTrue';
+};
+
+
+goog.inherits(os.ui.filter.op.IsTrue, os.ui.filter.op.Op);
+
+
+/**
+ *
+ * @inheritDoc
+ */
+os.ui.filter.op.IsTrue.prototype.getEvalExpression = function(v, literal) {
+  return '(!(typeof ' + v + '==="undefined"||' + v + '==null||' + v + '.length==0)&&(' +
```

Comment:
Same comment here. The expression `v === true || String(val).toLowerCase() === 'true' || v === 1` should cover all cases, and return false for all other values.